packages/babel-plugin-transform-es2015-arrow-functions/README.md

```diff
@@ -14,10 +14,18 @@ $ npm install babel-plugin-transform-es2015-arrow-functions
 
 **.babelrc**
 
-```json
+```js
+// without options
 {
   "plugins": ["transform-es2015-arrow-functions"]
 }
+
+// with options
+{
+  "plugins": [
+    ["transform-es2015-arrow-functions", { "spec": "true" }]
```

Comment:
Should this (and the other occurences of string `"true"`) be a plain boolean `true` instead?